### ISSUE TITLE
some suggestions

### ISSUE THREAD

**Issue by peow** [WerDei/Server-Hats#10]
- make the ghast tear render like a real tear

![image](https://github.com/WerDei/Server-Hats/assets/105888751/7a85deec-3e8b-4881-82d1-15a4656500d0)
(how it render now)

![image](https://github.com/WerDei/Server-Hats/assets/105888751/1e5539f5-c187-455e-9219-c05b6d6607ef)
(how i imagine)
*Im assuming you can control how big the item is rendered.

- make compat with trinket slots, then we can use an helmet and the hat
![image](https://github.com/WerDei/Server-Hats/assets/105888751/0009ef82-fab7-498d-80c4-5eb2499db4f4)

- i dont know if im dumb but the conduit dont change anything when i wear it

this mod is really great, if you gonna use some of these suggestion, please make an update to 1.20.1

**Comment by WerDei** [comment]
Hi, thank you so much for your suggestions! I have some bad news, and some good news.

Keep in mind that all of the item hat models that you see are added by Minecraft itself. This mod just adds an ability to equip these items without a command.

Server Hats was designed to be server-side only, meaning clients don't need it installed. For this reason, adding new item models is a bit tricky - clients would either need to install a resourcepack manually, or server resourcepack would need to be configured (which is a bit of a hassle for the owner). 

Trinket support has a simillar problem - the mod was made to be vanila-compatible, meaning vanilla clients should be able to connect. Adding trinkets to the mix would undermine that, since it needs to be installed on the client.

That said, **I've been thinking about creating another version of this mod** that would be based on Trinkets.
- Since It would be needed on the client anyway, it would make adding new item models very easy (apart from making them, of course)
- To make Server Hats have all the settings it has, I added a lot of funky code that, frankly, I am now a bit afraid to touch. I don't want introduce *another* game-breaking bug. New mod would be a fresh start without all of this spaghetti.
- To add to that, most of the custom things I wrote would instead be handled by Trinkets, so the new mod would be small and easy to support.

The only issue with making a new mod is, well, it requires making a new mod. I'm not sure if I'll have the time for that in the near future, but I will keep this on my mind!

Sorry for the wall of text, lol. I will now close this as "not planned", but when/if I create an alternative mod, I'll comment here to let you know!
